src/main.rs

```diff
@@ -9,16 +9,33 @@ use unicode_normalization::{is_nfc, UnicodeNormalization};
 const CONCURRENT_TASKS: usize = 32768;
 static SEMA: Semaphore = Semaphore::const_new(CONCURRENT_TASKS);
 
+/// jaso normalizes filenames to their Unicode NFC format in parallel
 #[derive(Parser, Debug)]
 #[clap(version, arg_required_else_help = true)]
 struct Args {
+    /// Follows symbolic links to directories.
+    ///
+    /// Note that current implementation of jaso allows infinite recursion due to cyclic symbolic
+    /// links.
     #[arg(long)]
     follow_directory_symlinks: bool,
+    /// Shows additional information, such as what files has been renamed.
+    ///
+    /// This option is useful for debugging or logging.
     #[arg(short, long)]
     verbose: bool,
+    /// Just indicates what would be renamed, without actually renaming files.
+    ///
+    /// This option is useful for checking if normalization is needed. This option implies
+    /// `--verbose` option.
```

Comment:
not 100% sure, but should we add a "the" article here?
```suggestion
    /// the `--verbose` option.
```